dwarfutils/pkg.yaml

```diff
@@ -0,0 +1,41 @@
+name: dwarfutils
+dependencies:
+  - stage: base
+  - stage: zlib
+  - stage: elfutils
+steps:
+  - sources:
+      - url: https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/dwarfutils/20210528-1/dwarfutils_20210528.orig.tar.gz
+        destination: dwarfutils.tar.gz
+        sha256: b8ba0ee9b70d2052d45272489d79bf456c4d342fc8c3bba45038afc50ec6e28b
+        sha512: e0f9c88554053ee6c1b1333960891189e7820c4a4ddc302b7e63754a4cdcfc2acb1b4b6083a722d1204a75e994fff3401ecc251b8c3b24090f8cb4046d90f870
+    prepare:
+      - |
+        tar -xvf dwarfutils.tar.gz --strip-components=1
+
+        ./configure --prefix=/usr --enable-shared
+    build:
+      - |
+        make
+    install:
+      - |
+        cd /tmp/build/0/libdwarf
```

Comment:
`/tmp/build/0` is a but of internal detail of the way `bldr` does things

I think it might be better to do:

````
pushd ./libdwarf
....
popd

pushd ./libdwarf/.libs
...
popd

...
```

Reply:
fixed